docs/BR.md

```diff
@@ -1324,50 +1324,42 @@ No stipulation.
 
 ### 4.9.9 On-line revocation/status checking availability
 
-The following SHALL apply for communicating the status of Certificates which include an Authority Information Access extension with an id-ad-ocsp accessMethod.
+The validity interval of an OCSP response is the difference in time between the `thisUpdate` and `nextUpdate` field, inclusive. For purposes of computing differences, a difference of 3,600 seconds shall be equal to one hour, and a difference of 86,400 seconds shall be equal to one day, ignoring leap-seconds.
```

Comment:
`nextUpdate` should not be counted in the validity interval of an OCSP response, because in section 3.2 of RFC6960 it is stated that `nextUpdate` must be in the future for the OCSP response to be valid

Reply:
Changing the way the validity interval is computed is outside the scope of this ballot. In this ballot, this line is simply moved from its previous location, its contents are not changed at all.

Reply:
This ballot could be a good opportunity to fix this as well

Reply:
I think it's unclear whether there's actually something to be fixed here, so I'd prefer not to bog this ballot down with that separate debate.

RFC 6960 has this to say:

> #### 2.4. Semantics of thisUpdate, nextUpdate, and producedAt
> nextUpdate: The time at or before which newer information will be available about the status of the certificate.

> #### 4.2.2.1 Time
> The thisUpdate and nextUpdate fields define a recommended validity interval.  This interval corresponds to the {thisUpdate, nextUpdate} interval in CRLs. Responses whose nextUpdate value is earlier than the local system time value SHOULD be considered unreliable.

Unfortunately, RFC 5280 has nothing to say about how clients should consume the `nextUpdate` field of CRLs, so there's no additional clarity to be found there.

So given the quotes above, we have the following problems:
1) The requirement in 4.2.2.1 is only a SHOULD, not a MUST.
2) The precedent set by the certificate's Validity Period being inclusive of the `notAfter` timestamp is that timestamps in X.509 documents represent the _whole_ time interval which could possibly be referenced by that timestamp, not the instant at the beginning of that interval. A `notAfter` timestamp of `2024-08-13 09:04:00` means that the certificate can be trusted throughout the whole `09:04:00` second, and only becomes untrusted when the clock rolls over to `09:04:01`. There's no indication that CRL and OCSP `nextUpdate` timestamps are to be treated any differently. If an OCSP response has a `nextUpdate` of `2024-08-13 09:04:00`, and that timestamp represents that _whole_ second and not just the instant at the beginning of that second, then a client whose clock says it is `09:04:00.5` is not looking at a "response whose nextUpdate value is earlier than the local system time value". It's looking at a response whose nextUpdate _contains_ the local system time, and is therefore under no obligation to treat the response as unreliable.

Look, I strongly dislike the interpretation of X.509 timestamps as being inclusive of their whole interval. I think it is unnecessarily pedantic and highly unintuitive. But the fact is that precedent for this interpretation is so well established that people decided to encode it directly into the BRs to prevent any future confusion. While I would be personally in favor of a ballot that does away with inclusive timestamps, that ballot should tackle OCSP, CRLs, and Certificates together. This ballot is not the right place to make that change.

Reply:
It is in section 3.2 of RFC 6960:
> [3.2](https://datatracker.ietf.org/doc/html/rfc6960#section-3.2).  Signed Response Acceptance Requirements
> 
>    Prior to accepting a signed response for a particular certificate as
>    valid, OCSP clients SHALL confirm that:
>
>    6. When available, the time at or before which newer information will
>       be available about the status of the certificate (nextUpdate) is
>       greater than the current time.

i.e. For an OCSP response to be valid the following inequality must hold: `nextUpdate > currentTime`
Unfortunately, this only applies to OCSP responses.

I don't mind if we don't change it. Just thought it would be nice to make this sentence more accurate.